### PR TITLE
[sdk/javascript] subtask: add helper functions 

### PR DESCRIPTION
**Pull request by gimre-xymcity**
bytesToBigInt and intFromBytes

### CODE REVIEW

File: sdk/javascript/test/utils/converter_spec.js [hunk 1]
```diff
@@ -181,4 +181,144 @@ describe('converter', () => {
...
+
+			canParseUnsigned(array, 8, BigInt('0xFEAA553399DDCCEE'));
+			canParseUnsigned(array, 4, BigInt('0x99DDCCEE'));
+			canParseUnsigned(array, 2, BigInt('0xCCEE'));
+			canParseUnsigned(array, 1, BigInt('0xEE'));
+		});
+
+		it('cannot parse too short array', () => {
+			// Act:
+			expect(() => converter.bytesToBigInt(new Uint8Array([0xaa, 0x55, 0xcc, 0x33, 0xfe, 0xdc, 0xba, 0x98]), 9)).to.throw();
```
Comment (Jaguar0625) [inline sdk/javascript/test/utils/converter_spec.js:213]: capitals ^^

File: sdk/javascript/test/utils/converter_spec.js [hunk 2]
```diff
@@ -181,4 +181,144 @@ describe('converter', () => {
...
+		});
+
+		it('can parse data as signed numbers', () => {
+			// Arrange: data in both arrays is aranged so there are positive and negative numbers
+			const positiveArray = [0x7E, 0x7C, 0xDD, 0x79, 0x33, 0x55, 0xAA, 0x7E];
+			const negativeArray = [0xEE, 0xCC, 0xDD, 0x99, 0x33, 0x55, 0xAA, 0xFE];
+
+			canParseSigned(positiveArray, 8, BigInt('0x7EAA553379DD7C7E'));
+			canParseSigned(negativeArray, 8, -BigInt(' 0x155aacc66223312'));
```
Comment (Jaguar0625) [inline sdk/javascript/test/utils/converter_spec.js:222]: capitals + spurious leading space in string

File: sdk/javascript/src/utils/converter.js [hunk 3]
```diff
@@ -110,6 +110,61 @@ const converter = {
...
+	},
+
+	/**
+	 * Converts byte stream to little-endian number.
+	 * @param {Uint8Array} input A uint8 array.
+	 * @param {numeric} size Number of bytes.
+	 * @param {boolean} isSigned Should number be treated as signed.
+	 * @returns {numeric|BigInt} Value corresponding to the input.
+	 */
+	intFromBytes: (input, size, isSigned = false) => {
```
[the remark this answers is not in the document]
Comment (gimre-xymcity) [reply sdk/javascript/src/utils/converter.js:147]: I think it's only avail in node

File: sdk/javascript/test/utils/converter_spec.js (hunk 1 above)
Comment (gimre-xymcity) [reply sdk/javascript/test/utils/converter_spec.js:213]: :rage3:

File: sdk/javascript/test/utils/converter_spec.js (hunk 2 above)
Comment (gimre-xymcity) [reply sdk/javascript/test/utils/converter_spec.js:222]: hmm, that spurious space waas there earlier to align it nicely, but now it doesn't make sense :/

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**